test/Serialization/Hash.purs

```diff
@@ -67,14 +67,14 @@ suite = do
     (isNothing $ scriptHashFromBech32 invalidBech32)
 
   scrh <- errMaybe "scriptHashFromBytes failed" $ scriptHashFromBytes
-    $ wrap
     $ hexToByteArrayUnsafe
         scriptHashHex
   let
     scrhB32 = scriptHashToBech32Unsafe "stake_vkh" scrh
     mScrhB32 = scriptHashToBech32 "stake_vkh" scrh
-    scrhBts = scriptHashToBytes scrh
-    scrhFromBytes = scriptHashFromBytes scrhBts
+    scrhBts = toBytes scrh
+    -- TODO: use fromBytes instead?
```

Comment:
Does it work? If so, yes.

Reply:
Currently my understanding is that `scriptHashFromBytes`, `ed25519KeyHashFromBytes`, `vrfKeyHashFromBytes`, and `vrfKeyHashToBytes` are all meant to be separate functions because `to/from_bytes` for hashes doesn't use CBOR, but just returns raw bytes instead, so for now I'm just leaving these functions alone.